backend/app/modules/access/auth_controller.py

```diff
@@ -301,3 +303,177 @@ def confirm_forgot_password(body: ConfirmForgotPasswordRequest,
                             })
 
     return {"message": "Password reset successful"}
+
+
+
+
+@router.post("/invite",
+             description="Invites a new user to application after creating a new account with user email and a temporary password in AWS Cognito.",
+             response_model=InviteResponse)
```

Comment:
Since the frontend is not expecting a response body on successful invitation, you can remove the `response_model=InviteResponse` code.